modules/geochemistry/paper.md

```diff
@@ -140,15 +139,47 @@ It is easy to use multiple sub-steps within (1) and/or (2), which may be necessa
+Spatially-varying geochemistry simulations use a large amount of memory since they store information about a complete geochemical system at each finite-element node (irrespective of the finite-element type).  On the other hand, ignoring transport, they are almost embarrasingly parallel, and can be solved efficiently using a large number of processors.  Even reactive-transport models scale well with multiple processors, since multi-component flow in porous media is essentially multiple coupled diffusion equations.  \autoref{fig:scaling_eg} shows an example of scaling in which an operator-split model of fixed size and fixed number of time-steps is run on multiple processors.  Further experiments involving memory usage, solver choices and cpu-scaling can be found in the [online documentation](https://mooseframework.org/modules/geochemistry/theory/compute_efficiencies.html).
 
 ![CPU time required to solve a reactive-transport simulation using the PorousFlow and geochemistry modules.\label{fig:scaling_eg}](joss_paper_scaling.png){ width=60% }
 
-# Example: cooling with feldspars
+# Building and using the code
+
+Building the geochemistry module is a two-step process:
+
+1. Download and install the entire MOOSE package.  Detailed instructions are at the [MOOSE website](https://mooseframework.inl.gov/getting_started/installation/index.html).  Even if only in geochemistry is of interest (without transport, solid mechanics, etc) MOOSE comes as a complete package, so needs to be installed in its entirety.  Depending on the computer setup, this can be straightfoward (on a personal Mac computer) or complicated (on an administered supercomputer).
+
+2. During the installation of MOOSE in part 1, only the "framework" will have been compiled.  To compile all the physics modules, including the geochemistry module, use the following instructions run from the command line:
+```
+cd ~/projects/moose/modules
+make
+cd ~/projects/moose/modules/geochemistry
+make
+```
+(If your computer has $N$ cores, the `make` process may be sped up by using the command `make -j`$N$ instead of simply `make`.)
+
+Check that the geochemistry module is correctly compiled using the following instructions:
+```
+cd ~/projects/moose/modules/geochemistry
+./run_tests
+```
+Virtually all the tests should run and pass.  Some may be "skipped" due to a particular computer setup (for instance, not enough threads).
```

Comment:
```suggestion
Virtually all the tests should run and pass.  Some may be "skipped" due to a particular computer setup (for instance, not enough threads). (If your computer has $N$ cores, the `run_tests` command may be sped up by using the command `./run_tests -j`$N$ instead of simply `./run_tests`.)
```

Reply:
thanks